### ISSUE TITLE
phar error: attempted to extract non-existent file or directory on location:update

### ISSUE THREAD

**Issue by 3likayed** [stevebauman/location#179]
Hello i'm using the last version of the package ^7.3.1 on windows
when calling the artisan command 
```
php artisan location:update
```
i get this phar exception 

```

   PharException

phar error: attempted to extract non-existent file or directory "phar://E:/latest/database/maxmind.tar.gz/GeoLite2-City_20240628\GeoLite2-City.mmdb" from phar "E:/latest/database/maxmind.tar.gz"

at vendor\stevebauman\location\src\Drivers\MaxMind.php:52
48▕
49▕         $directory = Str::afterLast($file->getPath(), DIRECTORY_SEPARATOR);
50▕
51▕         $relativePath = implode('/', [$directory, $file->getFilename()]);
➜  52▕         $archive->extractTo($storage->path('/'), $relativePath, true);
53▕
54▕         file_put_contents(
55▕             $this->getDatabasePath(),
56▕             fopen($storage->path($relativePath), 'r')

1   vendor\stevebauman\location\src\Drivers\MaxMind.php:52
PharData::extractTo("E:\latest\database\", "phar://E:/lat/database/maxmind.tar.gz/GeoLite2-City_20240628\GeoLite2-City.mmdb")

2   vendor\stevebauman\location\src\Commands\Update.php:34
Stevebauman\Location\Drivers\MaxMind::update(Object(Stevebauman\Location\Commands\Update))

```
this issue was mentioned before here #177 
Thanks,

**Comment by 3likayed** [comment]
pr #180

**Comment by stevebauman** [comment]
Thanks again for the PR @3likayed 🙏 

I've just released [v7.3.2](https://github.com/stevebauman/location/releases/tag/v7.3.2) that contains your patch.
